scapy/layers/all.py

```diff
@@ -7,15 +7,22 @@
 All layers. Configurable with conf.load_layers.
 """
 
+import __builtin__
 from scapy.config import conf
 from scapy.error import log_loading
-import logging
+import logging, importlib
+ignored = list(__builtin__.__dict__.keys()) + ["sys"]
 log = logging.getLogger("scapy.loading")
 
 __all__ = []
 
+
+def _validate_local(x):
```

Comment:
it's used each time we want to import a module. it replaces the "name not beginning by _" test, that already existed in those modules